### PR TITLE
fix(cssPrefix): fix logic that checks jQuery version

### PR DESCRIPTION
**Pull request by sculove**
## Issue
#196 

## Details
- fix logic that checks jQuery version.
- change Qunit 2.x style

## Preferred reviewers
@kishu

### CODE REVIEW

File: src/hook/cssPrefix.js [hunk 1]
```diff
@@ -27,7 +27,8 @@ eg.module("cssPrefix", ["jQuery", document], function($, doc) {
 	}
 
 	// run in jQuery 1.8.x below
-	if ($.fn && $.fn.jquery && $.fn.jquery.replace(/\./, "") >= "18") {
+	var matchTest = $.fn.jquery.match(/\d+\.\d+/);
+	if (!matchTest || matchTest.length <= 0 || parseInt(matchTest[0].replace(/\./, ""), 10) >= 18) {
```
Comment (netil) [inline src/hook/cssPrefix.js]:
```js
var matchTest = ($.fn.jquery.match(/^\d\.\d+/) || [])[0];

if (!matchTest || +matchTest.replace(/\D/,"") >= 18) {
	return;
}
```

Comment (sculove) [inline src/hook/cssPrefix.js]: good

File: test/unit/js/cssPrefix.test.js [hunk 2]
```diff
@@ -18,12 +18,12 @@ module("css", {
 
     jQuery.cssHooks = {};
   },
-  teardown : function() {
+  afterEach : function() {
```
Comment (netil) [inline test/unit/js/cssPrefix.test.js]: There is reason to leave empty function?

File: test/unit/js/cssPrefix.test.js [hunk 3]
```diff
@@ -37,23 +37,60 @@ test("When is not jQuery.cssHooks", function() {
     // When
 
     //Then
-    equal(method, false);
+    QUnit.equal(method, false);
```
Comment (netil) [inline test/unit/js/cssPrefix.test.js]: I thinkg it should be `assert.equal`
https://qunitjs.com/upgrade-guide-2.x/#replace-global-assertions-with-assert-arguments

Comment (sculove) [inline test/unit/js/cssPrefix.test.js]: you're right. I wonder how this test passed.
ok. I fit it!

File: test/unit/js/cssPrefix.test.js (hunk 2 above)
Comment (sculove) [inline test/unit/js/cssPrefix.test.js]: yes. ok

**Comment by netil** [comment]
LGTM

**Pull request merged**